DataFormats/Detectors/TOF/include/DataFormatsTOF/RawDataFormat.h

```diff
@@ -13,12 +13,31 @@
 /// @since  2019-12-18
 /// @brief  TOF raw data format
 
+/*
+
+   ALICE TOF Raw Data description: master file
+
+   Header file to describe TOF event fragment: defines, macros and data structures
+
+
+   @P. Antonioli / INFN-Bologna
+   March 2006
+
+   Dec. 2019 moved old CDH to new TDH, updated format for RUN3: new DRM2, new TRM format
+   Dec. 2019 agreed O2 format
+   20200324 fixed error on LTM_EVSIZE macro, use of __cplusplus directive
+   20200325 detField description
+   ToDo:
+   -- insert new fw upgrades in DRMH
+
+*/
+
 #ifndef O2_TOF_RAWDATAFORMAT
 #define O2_TOF_RAWDATAFORMAT
 
-#include <cstdint>
+#include <stdint.h>
 
-#ifndef O2_TOF_RAWDATAFORMAT_NONAMESPACE
+#ifdef __cplusplus
```

Comment:
why do you need this guard?

Reply:
this file is also being used for other purposes in our DCS machines.
There we compile with plain c compiler, so we need the guard to allow compilation there.